koptional-reactor-extensions/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'kotlin'
+apply plugin: 'org.junit.platform.gradle.plugin'
+apply from: '../gradle/publishing.gradle'
+
+dependencies {
+    compile libraries.koptional
+    compile libraries.reactor
+}
+
+dependencies {
+    testCompile libraries.spek
+    testCompile libraries.spekJunitPlatformEngine
```

Comment:
Please add `testCompile libraries.junitPlatformLauncher` as you did in the previous PR #8  :)